### DISCUSSION TITLE
tell server which GPU to use

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/MONAILabel#536] [Q&A] [answered]
For a local machine with multiple GPUs, how to tell the MONAI Label server which GPU and how many GPUs to use?

**Comment by SachidanandAlle** [comment]
You can use `CUDA_VISIBLE_DEVICES=1,2` at individual process level..

**Reply by hw-ju** [reply]
Thanks @SachidanandAlle! 
Shall I add the following to a script? Which python script shall I add it to?
`import os`
`os.environ["CUDA_VISIBLE_DEVICES"]=1,2`

**Comment by SachidanandAlle** [comment, marked as answer]
```
export CUDA_VISIBLE_DEVICES=1,2 
monailabel start_server -a <app_dir> -s <studies_dir>

```

**Reply by hw-ju** [reply]
Thanks a lot! @SachidanandAlle
